modular_chomp/code/game/jobs/job/noncrew.dm

```diff
@@ -23,13 +23,58 @@
 		-----Outsiders are expected to behave in accordance with Unauthorized Personnel SOP regardless of their IC knowledge.
 		-----Outsiders are not allowed to log-off with station key items (e.g. Captain's spare, station blueprints, nuclear authentication disk, bluespace harpoon, large quantities of station goods, etc). Please leave these items on station or with relevant crew.
 		-----We encourage outsiders to take on exploration content as a group, staff will not help you for any hardships of solo play.
-		-----Notice: The outsider role is relatively new; if you encounter bugs, please notify a staff member and avoid using exploits."}
-	alt_titles = list("Spacefarer" = /datum/alt_title/spacefarer)
+		-----Notice: The outsider role is relatively unpolished; if you encounter bugs, please notify a staff member and avoid using exploits."}
+	alt_titles = list(
+		"Tyr Local" = /datum/alt_title/tyr_local,
+		//"Thor Local" = /datum/alt_title/thor_local,
```

Comment:
I'm putting this here in case anyone decides to add Thor spawn points for outsiders. Would be very nice if someone did.